### DISCUSSION TITLE
How To Use Web-Components-V3 (fluent ui 2)

### DISCUSSION THREAD

**Discussion by carl-egge** [microsoft/fluentui#30424] [Q&A]
This might be a duplicate of my [discussion comment here](https://github.com/microsoft/fluentui/discussions/26542#discussioncomment-8217821) but since I think this is important I want to ask again.

Me an my team are trying to use Fluent UI 2 with other web frameworks than React (in our case Angular). We know that the branch [web-components-v3](https://github.com/microsoft/fluentui/tree/web-components-v3) tracks the progress with the implementation of Web Components for Fluent UI 2 and that a beta.6 version is already published.

Two things remain unclear to us and we would very much appreciate help with this!

1. Is there a roadmap or any indication when the web components v3, implementing fluent ui 2 for frameworks like Angular, will be finished and published to production?

2. How to use the beta.6 version in an Angular project? Since beta should be end-user testing we would very much love to use and test the new implementation of web components v3.

Thank you for any hints!
Carl

**Comment by carl-egge** [comment]
This Question also asks for similar help: https://github.com/microsoft/fluentui/discussions/30367#discussion-6121244

**Comment by carl-egge** [comment]
This issue has also an unanswered question regarding the roadmap: https://github.com/microsoft/fluentui/issues/29688#issuecomment-1813374564

**Comment by khmakoto** [comment]
FYI @chrisdholt

**Comment by miguelc1221** [comment]
yea seems "provideFluentDesignSystem" is missing in v3, happy to open up a PR when i have some time but not sure if its something being worked on already. My team is also waiting for an update on this = ). Thanks!

**Reply by chrisdholt** [reply]
Currently in process - there are still some changes coming and we need to get the single file script working. Will update as we get a PR out for “Getting Started”

**Reply by carl-egge** [reply]
@chrisdholt Do you have a specific timeline or deadline by which we can expect the "Getting Started" PR?

**Reply by miguelc1221** [reply]
just curious if there were any updates on this @chrisdholt. Thanks!
